kornia/feature/siftdesc.py

```diff
@@ -1,12 +1,14 @@
 from typing import Tuple
-import kornia
 import torch
 import math
 import torch.nn as nn
 import torch.nn.functional as F
-import numpy as np
+from kornia.filters import get_gaussian_kernel2d
+from kornia.filters import spatial_gradient
+from math import pi
```

Comment:
we have kornia.pi which already cast pi number into a torch tensor. We can discuss the use of it.

Reply:
fixed